bescort.lic

```diff
@@ -227,6 +227,7 @@ class Bescort
     ]
 
     args = parse_args(arg_definitions)
+    @flying_mount = get_settings.flying_mount
```

Comment:
💡 Looking at the rest of the file, I think the pattern has been to call `get_settings` within the specific travel function (e.g. `use_flying_mount`). I'm not a fan of that in general, but it does mean we don't have a bunch of instance vars for travel types that will never be used (in that specific execution of `bescort`).

What do you think about storing `get_settings` in an instance var here and then each place in `bescort` can just grab the N settings it needs? That could also be pushed off to a future PR, whether by you or someone else.

Reply:
Sounds good.